apps/docs/troubleshooting/extglossary.md

```diff
@@ -0,0 +1,98 @@
+# Glossary of Functions
+
+<sub>This portion of the guide aims to outline specific functions that are asked about the most by new users of PreMiD. It is not comprehensive, but it should give new users context into what they may be looking at.</sub>
+
+_Please note, this portion of the guide is always expanding. Alpha, Beta, and Release builds of PreMiD will continue to evolve the information contained._
+
+## PreMiD Extension Functions
+
+> [!NOTE]
+> There will be images contained that are from development versions of PreMiD. These will not always show all of the listed functions.
+
+![PreMiD Functions](/guide-images/gu-p1-pmsetting.png)
+
+**Heartbeat**
+
+- Heartbeat is a diagnostic tool. In _future versions of PreMiD_ this will show some fancy information and statistics.
+
+**Developer Mode**
+
+- A diagnostic and development tool. Most users will never use this. Toggling this on will open the websocket port of PreMiD.
+
+**Sync Language**
+
+- This option allows for the user to mandate their language settings be set between all presences based on the choice made in the PreMiD application.
+- If this is not selected, the user may mandate different languages for separate presences.
+
+**Use Playing Status**
+
+- _[Soon to be depreciated.]_ This tells PreMiD to use "Playing" status over use "Listening" status or use another type of action.
+- The extension is currently in the process of being updated. When this happens, the function will no longer exist-- as it is able to be controlled by a presence rather than the overall application.
+
+**Prefer App**
+
+- _[Soon to be depreciated.]_ When PreMiD utilized an application as well as an extension bundle, this would notify the extension of the application being installed on the machine, sending information to the local machine (or LocalHost) to transmit data to the application. That information would then be pushed directly to Discord. Due to Discord development and policy changes, there is unfortunately, no longer an application.
+- The extension is currently in the process of being updated. When this happens, the function will no longer exist.
```

Comment:
These don't exist

Reply:
>  _[Soon to be depreciated.]_

 We still have users on the old version.
 Can be removed at a later time.